src/main/java/com/baloise/incubator/argonaut/application/github/GitHubWebhookRestController.java

```diff
@@ -66,41 +67,43 @@ public void handleGitHubWebhookEvent(@RequestBody String data, @RequestHeader(GI
         }
     }
 
-    private void handlePullRequestEvent(GHEventPayload.PullRequest pullRequest) {
-            switch (pullRequest.getAction()) {
-                case "closed": {
-                    LOGGER.info("PR CLOSED Event");
-                    break;
-                }
-                case "opened": {
-                    LOGGER.info("PR OPENED Event");
-                    break;
-                }
-                default: {
-                    LOGGER.info("Unhandled GitHub Pull Request Event Action: {}", pullRequest.getAction());
-                }
+    private void handlePullRequestEvent(GHEventPayload.PullRequest ghPullRequest) {
+        switch (ghPullRequest.getAction()) {
+            case "closed": {
+                LOGGER.info("PR CLOSED Event");
+                break;
             }
+            case "opened": {
+                PullRequest pullRequest = createPullRequest(ghPullRequest.getNumber(), ghPullRequest.getRepository(), ghPullRequest.getPullRequest());
+                pullRequestCommentService.createPullRequestComment(new PullRequestComment("This PR is managed by **[Argonaut](https://github.com/baloise-incubator/argonaut).** \n\n You can use the command `/ping` as pull request command to test the interaction in a comment. \n\n You can use the command `/deploy` to deploy this branch to it's preview environment (after build is successfull). \nYou can use the command `/promote` to promote this branch to production.", pullRequest));
+                LOGGER.info("PR OPENED Event");
+                break;
+            }
+            default: {
+                LOGGER.info("Unhandled GitHub Pull Request Event Action: {}", ghPullRequest.getAction());
+            }
+        }
     }
 
     private void handlePushEvent(GHEventPayload.Push push) {
         LOGGER.info("REPO PUSH Event");
     }
 
-    private void handleIssueCommentEvent(GHEventPayload.IssueComment issueComment) {
+    private void handleIssueCommentEvent(GHEventPayload.IssueComment issueComment) throws IOException {
         switch (issueComment.getAction()) {
             case "created": {
                 String commentText = issueComment.getComment().getBody();
```

Comment:
Violation of Law of Demeter, maybe a method getCommentBody() can solve it.

Reply:
Das Objekt ist aus einem SDK, ned vo mir ;-)